ocfweb/lab_reservations/mail_templates/officer_notification.jinja

```diff
@@ -0,0 +1,9 @@
+Hello all,
+
+The OCF has received a lab reservation request!
+
+The request was issued by {{request.real_name}}
+({{request.contact_email}}) on behalf of the student group
+{{request.student_group}}. They wish to use our lab from {{request.starttime}}
+until {{request.endtime}} on {{request.date}}. Their stated reason is:
+{{request.reason}}
```

Comment:
(not blocking)

This is sort of nit-picky, but I think it's usually easier for people to parse form emails like this when the fields are just put in a list (similar to what we do for vhost requests). For example:


> Lab Reservation Request:
> - Submitter: *Chris Kuehl*
> - Email: *ckuehl@ocf.berkeley.edu*
> - Student Group: *ckuehl appreciation club*
> - Time: *6pm to 9pm*
> - Date: *2018-02-18*
>
> Request Reason:
> *details*

I don't really see the point of wrapping a machine-generated email in superfluous English that just needs to be parsed out.